src/KubernetesClient/Versioning/KubernetesVersionComparitor.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace k8s.Versioning
+{
+    public class KubernetesVersionComparer : IComparer<string>
```

Comment:
`System.Version`?
you may want to reuse it

Reply:
> is this to support api call to lower version k8s server?

Yes, there's an open PR for creating and installed CRDs from me. I've ran into this problem when installing into my test cluster which is actually using an older version of CRDs. I'm refactoring that code to be based on this one, and automatically do conversion based on server version. I'm partly blocked on that one until we figure out what to do with https://github.com/kubernetes-client/csharp/issues/417